Lib/test/test_zipfile.py

```diff
@@ -2431,6 +2440,46 @@ def build_abcde_files():
     return zf
 
 
+def build_abcdef_files():
```

Comment:
I wasn't happy with the name `abcde` and now I'm even less happy to see that concept proliferate. If possible, I'd like to create a single example that represents the full breadth of supported files. Or if that's not possible, these fixtures should probably have more meaningful names.

Reply:
I can change function (``build_abcde_files``) to be ``build_files`` and make it a generator where all three types of files are returned one by one. The problem I will face with doing that is that all tests are dependent on the type of zipfile and its structure, so one test will not work for all file types. Do you have any thoughts or suggestions on this?

Reply:
Another thought is to change the method ``zipfile_abcde`` to add be called zipfile_type with an added parameter which would pass in the string for file name. Based on the filename we can then yield the file?

Reply:
I want to move away from that naming altogether. I chose `abcde` because it was not important what it was named (it was the only thing). Now that we have new variants (although I'm not sure we need them), we need more meaningful names that describe the purpose of the zipfiles that the function is generating.